recipes/xerces-c/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+	export CC=clang
+	export CXX=clang++
```

Comment:
I think this is ok.

Mac is basically where I do all of my development (though I do a fair bit on Linux too) and I moved all Mac components of my recipes to use `clang` some time ago and this has worked pretty nicely. Later version of Mac don't really have `gcc` (it is a fancy alias to `clang` and some other special things going on depending on your OS version) so using `clang` ends up being the right way to go. Also, for really old Macs that still have `gcc` it is version 4.2.1 (ancient) so we really should be avoiding it if at all possible. This also means C++11 support, for instance, is only possible with `clang`. I used this strategy with Boost, as well. Though less overtly.